dts/bindings/clock/fixed-clock.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2019 Derek Hageman <hageman@inthat.cloud>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Generic fixed rate clock provider
+version: 0.1
+
+description: >
+    This is a representation of a generic fixed rate clock provider.
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "fixed-clock"
+
+    label:
```

Comment:
Do we need label?

Reply:
See #15585 again, the label is used to construct the clock tree: it's used to compare the input clock source (`..._CLOCK_CONTROLLER`) with the labels of the other clocks to determine the correct HAL constants to use to select the input clock.

Reply:
Ok, will move the conversation of to #15585, but I think `label` isn't what we should be using.  Probably should be `clock-names` or `clock-output-names`